src/app/pages/MarginTradePage/index.tsx

```diff
@@ -18,7 +18,6 @@ import { Theme, TradingChart } from '../../components/TradingChart';
 import { OpenPositionsTable } from './components/OpenPositionsTable';
 import { useIsConnected } from '../../hooks/useAccount';
 import { TradingHistory } from './components/TradingHistory';
-import { NotificationForm } from '../../components/NotificationForm/NotificationFormContainer';
```

Comment:
is `NotificationFormContainer` component used somewhere else? Maybe it can be deleted